maths/modular_inverse.nim

```diff
@@ -0,0 +1,65 @@
+## Modular inverse
+
+import std/[options, math]
+
+
+func euclidHalfIteration(inA, inB: Positive): (Natural, int) =
```

Comment:
```suggestion
func euclidHalfIteration(inA, inB: Positive): (gcd: Natural, coeff: Natural) =
```
I prefer naming the return values. This is visible in the documentation and thus in the API exposed to the user.
The names of the resulting variables are not displayed otherwise.

Reply:
Partially done in b7e3dcd85f02c51339f22da1a10e2ad538e656da. As far as I can tell, the suggestion does not have a correct syntax.

Reply:
Thanks. I constantly switch between languages, and I have trouble memorizing the tuple syntax for procedures which I seldom use.